src/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs

```diff
@@ -243,15 +254,23 @@ public void RootId()
         public void StartStopWithTimestamp()
         {
             var activity = new Activity("activity");
-            Assert.Throws<InvalidOperationException>(() => activity.SetStartTime(DateTime.Now));
+            Assert.Equal(activity.StartTimeUtc, default(DateTime));
 
-            var startTime = DateTime.UtcNow.AddSeconds(-1);
+            activity.SetStartTime(DateTime.Now);    // Error Does nothing because it is not UTC
+            Assert.Equal(activity.StartTimeUtc, default(DateTime));
+
+            var startTime = DateTime.UtcNow.AddSeconds(-1); // A valid time in the past that we want to be our offical start time.  
             activity.SetStartTime(startTime);
 
             activity.Start();
-            Assert.Equal(startTime, activity.StartTimeUtc);
+            Assert.Equal(startTime, activity.StartTimeUtc); // we use our offical start time not the time now.  
+            Assert.Equal(activity.Duration, TimeSpan.Zero);
+
+            Thread.Sleep(35);
```

Comment:
Is there any significance to this very specific value of `35`?  Or just an arbitrary number that's likely a few quantums?